app/src/main/AndroidManifest.xml

```diff
@@ -12,15 +12,15 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.spendmanager.demo"
+        android:theme="@style/Theme.SpendManager.Demo"
         tools:replace="android:allowBackup">
         <activity android:name=".LauncherActivity" />
-        <activity android:name=".direct.DirectJavaActivity" />
-        <activity android:name=".direct.DirectKotlinActivity" />
-        <activity android:name=".subclass.SubclassKotlinActivity" />
-        <activity android:name=".subclass.SubclassJavaActivity" />
-        <activity android:name=".fragment.FragmentKotlinActivity" />
-        <activity android:name=".fragment.FragmentJavaActivity" />
+        <activity android:name=".direct.DirectJavaActivity" android:theme="@style/Theme.Sensibill" />
+        <activity android:name=".direct.DirectKotlinActivity" android:theme="@style/Theme.Sensibill" />
```

Comment:
The `direct` integration classes don't need the theme set

Reply:
removed